ToDoApp/ToDoApp/ToDoListView.swift

```diff
@@ -41,6 +42,25 @@ struct ToDoListView: View {
                         }
                     }
                     .listStyle(.plain)
+                    .padding(.bottom, addButttonSize)
+                }
+                Button( action: {
+                    viewModel.addButtonTapped()
+                }, label: {
+                    Image(systemName: "plus")
+                        .frame(minWidth: addButttonSize, minHeight: addButttonSize)
+                        .foregroundColor(.white)
+                        .background(Color(.primaryButton))
+                        .clipShape(Circle())
```

Comment:
<img width="257" alt="image" src="https://user-images.githubusercontent.com/24592433/207405927-a7c569d1-5aa2-4c98-9478-37fcfde36803.png">
trailing padding is missing as well